docs/yaml/02-config.yaml

```diff
@@ -93,3 +93,7 @@ data:
     # The path to find custom HTML templates
     # Env var: GANGWAY_CUSTOM_HTTP_TEMPLATES_DIR
     #customHTMLTemplatesDir: /custom-templates
+
+    # The path to find the CA bundle for the IdP, optional. Used to configure kubectl.
+    # Env var: GANGWAY_IDP_CA_PATH
+    # idpCAPath: "/cacerts/idpca.cert"
```

Comment:
[configuration doc](https://github.com/heptiolabs/gangway/blob/09b8ef7/docs/configuration.md) should be updated also.